c2corg_api/views/document.py

```diff
@@ -296,6 +296,35 @@ def _put(
 
         return {}
 
+    def _delete(self, clazz, schema, clazz_locale=None):
+        if not self.request.has_permission('moderator'):
+            raise HTTPForbidden('No permission to delete the document')
+        id = self.request.validated['id']
+
+        # get the current version of the document
+        document = self._get_document(clazz, id, clazz_locale=clazz_locale)
+        lang = document.locales[0].lang
+
+        associations = doc_associations.get_associations(document, lang, False)
```

Comment:
OK thanks, but anyway we need to get the list of associations in order to update their ``cache_versions`` when the document has actually been removed, don't we?